nestkernel/archiving_node.h

```diff
@@ -50,44 +50,32 @@ class ArchivingNode : public StructuralPlasticityNode
 {
 public:
   /**
-   * \fn ArchivingNode()
-   * Constructor.
+   * Constructor
    */
   ArchivingNode();
 
   /**
-   * \fn ArchivingNode()
-   * Copy Constructor.
+   * Copy Constructor
    */
   ArchivingNode( const ArchivingNode& );
 
   /**
-   * \fn double get_K_value(long t)
-   * return the Kminus (synaptic trace) value at t (in ms). When the trace is
+   * Return the Kminus (synaptic trace) value at t (in ms). When the trace is
```

Comment:
If I am not entirely mistaken, the same rules apply to doxygen comments as to Python docstrings: the first line should give a short summary of what a function does, followed by an empty line and then, if relevant, details. So here 
```c++
/**
 * Return the Kminus (synaptic trace) value at t (in ms). 
 * 
 * When the trace is ...
```

Reply:
Added a new line after first line description in comments